File: Java/UberVan.java
package Java;

import java.util.ArrayList;
import java.util.Map;

public class UberVan extends Carro {

    Map<String, Map<String,Integer>> tipoCarroAceptado;
    ArrayList<String> materialAsiento;

    public UberVan(String licencia, Persona conductor, 
    Map<String, Map<String,Integer>> tipoCarroAceptado,
    ArrayList<String> materialAsiento){
        super(licencia, conductor);
        this.tipoCarroAceptado = tipoCarroAceptado;
        this.materialAsiento = materialAsiento;     
    }

}

File: Python/uberVan.py
from carro import Carro

class UberVan(Carro):

    tipoCarroAceptado = []
    materialAsiento = []

    def __init__(self, licencia, conductor, materialAsiento, tipoCarroAceptado):
        super.__init__(licencia, conductor)
        self.materialAsiento = materialAsiento
        self.tipoCarroAceptado = tipoCarroAceptado

